ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -633,6 +633,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-nightly-sts
+- as: e2e-osd-ccs-nightly-aws
```

Comment:
`nightly` is not needed here.  The file name already indicate that.

Reply:
Ok, I will remove the word `nightly`.